src/patterns/dosdp-patterns/cellHasPlasmaMembranePartX.yaml

```diff
@@ -0,0 +1,50 @@
+pattern_name: cellHasPlasmaMembranePartX
+pattern_iri: http://purl.obolibrary.org/obo/cl/cellHasPlasmaMembranePartX
+
+description: A cell type that is characterized by a plasma membrane part, such as a cilium or receptor.
```

Comment:
Add notes that this is only good for things defined by a single PM receptor. What is our strategy for cardinality>1 @matentzn